test/system-ca/test-native-root-certs-env.mjs

```diff
@@ -0,0 +1,56 @@
+// Env: NODE_USE_SYSTEM_CA=1
```

Comment:
FYI @pmarchini when using this feature (thanks for implementing it by the way) I noticed that it does not yet have integration in `test/common/index.js` like the one for `Flags` so that when you run the test directly with `node /path/to/test.js`, the process would not automatically pause the evaluation and spawn a child process to run itself with the environment variable instead.

https://github.com/nodejs/node/blob/0fd1ecded6476378b207cacc8fb4e2f1698830c6/test/common/index.js#L90-L96